### PR TITLE
Add section about Self-Service app.

### PR DESCRIPTION
**Pull request by chrisequalsdev**
Resolves: #806

### CODE REVIEW

File: _scicomputing/access_credentials.md [hunk 1]
```diff
@@ -13,9 +13,17 @@ If one of your collaborators requires access to the Fred Hutch network you can s
 
 Please see the Service Desk site on CenterNet for more information about [HutchNet ID](https://centernet.fredhutch.org/cn/u/center-it/help-desk.html) including password rotation, etc.
 
+## SciComp Self-Service
+
+[SciComp Self-Service](https://scicomp-self-service.fredhutch.org/) allows you to gain access to some resources without submitting a ticket.  You can currently use this to do the following:
+
+- Add a new member to your lab group 
+- Enable cluster access for yourself under your lab's Slurm account
+- Check to see if you have the correct access needed to complete the Cluster 101 course
```
Comment (vortexing) [inline _scicomputing/access_credentials.md:22]: Can you put in the link to the bookdown course for this cluster 101 course?  https://hutchdatascience.org/FH_Cluster_Guide/

File: _scicomputing/access_credentials.md [hunk 2]
```diff
@@ -13,9 +13,17 @@ If one of your collaborators requires access to the Fred Hutch network you can s
...
+
 ## Accessing Slurm Clusters
 
-To use Slurm clusters like _gizmo_ you also need to have your HutchNet ID associated with a PI account.  Errors similar to "Invalid account or account/partition" typically indicate that this association hasn't been set up.  Contact Scientific Computing to have this corrected.
+To use Slurm clusters like _gizmo_ you also need to have your HutchNet ID associated with a PI account.  Errors similar to "Invalid account or account/partition" typically indicate that this association hasn't been set up.  Contact Scientific Computing to have this corrected or use [SciComp Self-Service](#scicomp-self-service).
```
Comment (vortexing) [inline _scicomputing/access_credentials.md:26]: I think we'd been trying to use backticks for gizmo/rhino. so `gizmo`?

**Review by vortexing — CHANGES_REQUESTED** [review]
This looks cool.  I think when this gets merged in you all should post in Slack about the new tool so people see it and know it exists!

File: _scicomputing/access_credentials.md (hunk 1 above)
Comment (chrisequalsdev) [reply _scicomputing/access_credentials.md]: done!

File: _scicomputing/access_credentials.md (hunk 2 above)
Comment (chrisequalsdev) [reply _scicomputing/access_credentials.md]: done!

**Review by vortexing — APPROVED** [review]

**Pull request merged**